cmd/destroy.go

```diff
@@ -0,0 +1,115 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package cmd defines command line utilities for ghpc
+package cmd
+
+import (
+	"fmt"
+	"hpc-toolkit/pkg/config"
+	"hpc-toolkit/pkg/modulewriter"
+	"hpc-toolkit/pkg/shell"
+	"os"
+	"path/filepath"
+
+	"github.com/spf13/cobra"
+)
+
+func init() {
+	artifactsFlag := "artifacts"
```

Comment:
nit. Is there a point of using local `artifactsFlag` and `autoApproveFlag` instead of string literals in place?

Reply:
artifactsFlag gets used twice (albeit on consecutive lines) so I'd rather tie those two usages together. I'll remove autoApproveFlag and convert function to `BoolVar` instead of `BoolVarP` (due to no short version of flag).